guides/common/modules/snip_host-registration-steps.adoc

```diff
@@ -42,11 +49,18 @@ You can use the ID of any {SmartProxyServer} that you configured for host regist
 Include the `--force` option to register a host that has been previously registered to a {SmartProxyServer}.
 endif::[]
 . Connect to your host using SSH and run the registration command.
-ifdef::client-content-dnf[]
+ifdef::satellite[]
 . Check the `/etc/yum.repos.d/redhat.repo` file and ensure that the appropriate repositories have been enabled.
 endif::[]
+ifndef::satellite[]
```

Comment:
Works for me! Feel free to create a section for orcharhino if you also want it displayed in a similar fashion.